src/stores/Entities/hooks.ts

```diff
@@ -35,17 +35,21 @@ export const useEntitiesStore_capabilities_readable = () => {
 //     );
 // };
 
-export const useEntitiesStore_capabilities_adminable = () => {
+export const useEntitiesStore_atLeastOneAdminTenant = () => {
```

Comment:
What do you think about `adminExists` or `adminTenantExists` as an alternative term? The current name is _fine_ as-is but I know this convention is used elsewhere for existence checks.

Reply:
With the change in how we store things I think we can also change the approach to naming. So potentially good for future.

I think `adminExists` was one I ruled out because I'd worry we would add other "admin" things (like `adminUser`) that we might also need to check.